includes/class-wcpdf-admin.php

```diff
@@ -75,6 +75,9 @@ public function __construct()	{
 		add_filter( 'woocommerce_rest_prepare_report_orders', array( $this, 'add_invoice_number_to_order_report' ) );
 		add_filter( 'woocommerce_report_orders_export_columns', array( $this, 'add_invoice_number_header_to_order_export' ) );
 		add_filter( 'woocommerce_report_orders_prepare_export_item', array( $this, 'add_invoice_number_value_to_order_export' ), 10, 2 );
+
+		add_filter( 'heartbeat_received', array( $this, 'handle_heartbeat_received' ), 10, 2 );
```

Comment:
Do you mean you had the metabox without any data?

It's weird because the [`document-data-metabox.php`](https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/757/files#diff-0d342e77085acd3923be230ea36e354d74d4232e687d0bc4f8aafe3eecc3fe0aR807) should be loaded on the page load anyway and display something.

Could you help me replicate it?

Reply:
Please let me know if this has been fixed by this commit: https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/pull/757/commits/acc87cb0ddd1de0aac3c2d489192ef09d5f49bdc.